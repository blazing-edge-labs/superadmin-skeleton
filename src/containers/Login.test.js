import React from 'react'
import { shallow } from 'enzyme'

import Login from './Login'

// TODO: find out how to render(test) this (redux)(redux-form)connected component

describe('Login', () => {
  const login = shallow(<Login/>)

  it('renders properly', () => {
    expect(login).toMatchSnapshot()
  })
})
