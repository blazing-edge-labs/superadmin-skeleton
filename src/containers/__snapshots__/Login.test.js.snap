// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders properly 1`] = `
<Connect(Form(Connect(Login)))
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="signIn"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldValidate={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
/>
`;
